docs/single-bench.md

```diff
@@ -81,8 +81,6 @@ Notes:
 - After the site is ready the username is `Administrator` and the password is `$ADMIN_PASSWORD`
 - The local deployment is for testing and REST API development purpose only
 - A complete development environment is available [here](../development)
-- The site names are limited to patterns matching \*.localhost by default
```

Comment:
these 2 point do have things to convey.

1. If nothing is configured, by default `.localhost` will work in browser
2. If hostnames and ip(s) are added to `/etc/hosts` it can work with any domain name.

Why? because frappe checks for `Host` header to serve one of its site from multitenant bench.

Reply:
resolution depends on the local resolver. This .localhost thing is a special config that may or may not work depending on the system. I am aware of  RFC 2606 (BCP 32) - nevertheless providing SITE_NAME="foo.localhost" as this bit suggests does not work. 

Beyond that it is probably best to let the user figure out how to resolve their host names instead of explaining name resolution as part of the erpnext docs.